docs/developers.rst

```diff
@@ -28,6 +28,7 @@ Information on specific functions, classes, and methods.
 .. toctree::
    :glob:
 
+   api/eddymotion.cli
```

Comment:
This will have little effect. Please add the [sphinxarg extension](https://github.com/nipreps/mriqc/blob/3fe90466f34a16cb8d575a88c260db39b561bb7d/docs/source/conf.py#L43) to generate documentation for this automatically.

How to use it (MRIQC example):

https://github.com/nipreps/mriqc/blob/3fe90466f34a16cb8d575a88c260db39b561bb7d/docs/source/running.rst#L73-L79

And what it renders:

https://mriqc.readthedocs.io/en/latest/running.html#command-line-interface

Reply:
I added the extension but I'm not entirely sure what you're expecting based on the example you provided. If you're suggesting that I add documentation about the CLI in the user documentation, I believe it would be more appropriate to include that in PR #169.

Reply:
My suggestion is to have automated documentation created from the command line parser with the module I suggest, which results in something of this sort: https://mriqc.readthedocs.io/en/latest/running.html#command-line-interface As shown in the link, it just generates the help of the command and formats it properly.

 #169 is manually-written documentation, so it is more of a user guide where examples are shown.

Reply:
BTW - I think GitHub was too smart with the first link: https://github.com/nipreps/mriqc/blob/3fe90466f34a16cb8d575a88c260db39b561bb7d/docs/source/running.rst?plain=1#L73-L79

If it doesn't switch to "code" mode, please hit that switch -- otherwise you can't see the example.

Reply:
I've added the automatically generated CLI documentation into a new file named running.rst (https://github.com/nipreps/eddymotion/pull/157/commits/8e30d72e1f248eeeae54161057fcfaa9c2173538). However, I'm still wondering if it's better to move it to the same file as in PR #169, usage.rst, since it primarily explains how to run the code. It could be a separate subsection within the same file (in a separate PR from #169 to prevent confusion with the manually written documentation).